labman/gui/static/js/plateViewer.js

```diff
@@ -359,12 +359,10 @@ PlateViewer.prototype.getActiveStudy = function () {
   studyID = get_active_studies().pop();
 
   if (studyID === undefined) {
-    alert('Please select a study; plating samples without selecting a study ' +
-          'is disallowed');
-    return;
+    return 0;
```

Comment:
Would be good to document this behavior in the docstring of this method, and state the implication.